demo-simple/src/main/java/com/novoda/downloadmanager/demo/MainActivity.java

```diff
@@ -119,7 +119,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private final MigrationCallback migrationCallback = migrationStatus -> databaseMigrationUpdates.setText(migrationStatus.status().toRawValue());
 
-    private final View.OnClickListener startMigrationOnClick = v -> downloadMigrator.startMigration(migrationCallback);
+    private final View.OnClickListener startMigrationOnClick = v -> downloadMigrator.startMigration("downloads.db", migrationCallback);
```

Comment:
is this param mandatory?

Reply:
Yes, we need to specify the db filename

Reply:
I saw that in the v1 of the download manager we were using the metadata `com.novoda.downloadmanager.DatabaseFilename` to specify from a client perspective the db filename... we might want to use that one even now, in that case replacing the DB name param here